modules/core/notification/farm_notification.routing.yml

```diff
@@ -0,0 +1,7 @@
+farm_notification.settings:
+  path: '/farm/settings/notification'
+  defaults:
+    _form: '\Drupal\farm_notification\Form\NotificationSettingsForm'
+    _title: 'Notification settings'
+  requirements:
+    _permission: 'administer farm notification'
```

Comment:
I believe we need to add this to `managed_role_permissions` like here: https://github.com/farmOS/farmOS/blob/fc77c86abe6b142702e7b103cd85e0c207333b79/modules/core/data_stream/modules/notification/data_stream_notification.managed_role_permissions.yml#L3